pkg/service/repair/service_repair_integration_test.go

```diff
@@ -1784,9 +1788,36 @@ func TestServiceRepairIntegration(t *testing.T) {
 		ctx, cancel := context.WithCancel(context.Background())
 		defer cancel()
 
+		// Check small_table_optimization support
+		ni, err := h.Client.AnyNodeInfo(ctx)
+		if err != nil {
+			t.Fatal(err)
+		}
+		support, err := ni.SupportsRepairSmallTableOptimization()
+		if err != nil {
+			t.Fatal(err)
```

Comment:
Why ? Isn't it hidden assuption that all the envrionments that we use to test against supports smallTableOptimization ?
I suggest to use t.Skip("explanation") here

Reply:
This function returns error only when there is a problem with parsing Scylla version (so it should never happen). When `small_table_optimization` is not supported, this function returns `false, nil`. We can't skip such a case, because SM should work fine with Scylla versions that does not support  `small_table_optimization`.